integreat_cms/api/v3/socialmedia_headers.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module includes functions related to the pages API endpoint.
+"""
+import logging
+from html import unescape
+
+from django.conf import settings
+from django.shortcuts import render
+from django.utils.html import strip_tags
+from django.utils.translation import activate
+from django.utils.translation import gettext as _
+
+from ...cms.models import PageTranslation
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def socialmedia_headers(request, region_slug, language_slug, path):
+    """
+    Retrieves all children for a single page
+
+    :param request: The request that has been sent to the Django server
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: Slug defining the region
+    :type region_slug: str
+
+    :param language_slug: Code to identify the desired language
+    :type language_slug: str
+
+    :param path: Should be a page translation slug
+    :type path: str
+
+    :return: HTML meta headers required by social media platforms
+    :rtype: ~django.template.response.TemplateResponse
+    """
+    region = request.region
+    page_translation = PageTranslation.search(region, language_slug, path).first()
+    activate(language_slug)
+
+    if page_translation:
+        excerpt = (
+            unescape(strip_tags(page_translation.content))[:100]
+            if page_translation
+            else None
+        )
+        title = page_translation.title
+    else:
+        excerpt = _(
+            "hallo aschaffenburg is your digital companion for the town of Aschaffenburg. Here"
+            " you can find local information, advice centres and services."
+            if region == "hallo"
+            else "Your local information guide with over 100 regions in Germany."
+        )
+        title = "Local Information for You"
```

Comment:
Instead of hardcoding such a fallback value, we'd probably want to implement #402, right?

Reply:
Yes, we could indeed just add a new region property, if we can agree that we support only a single language for the time being.

Reply:
Ah hmm yeah forgot about the translations... I guess translations are pretty important in the mid-term? Then I'm fine with using the hardcoded approach now and build a good solution with own translation models in the next weeks/months...

Reply:
Theoretically we could include the `<html>` tag in the endpoint. This would enable translations for the meta description as well.

Reply:
I think Nginx does not care about which part of the HTML document we include. I have no idea how we would dynamically set a header though ;)